app/jobs/risc_delivery_job.rb

```diff
@@ -49,17 +54,20 @@ def perform(
       )
     end
   rescue *NETWORK_ERRORS, RedisRateLimiter::LimitError => err
-    raise err if !inline?
```

Comment:
To expand on my above comment, the issue is the job doesn't log a failure on retries, but only when it fails on the final retry. If we could check if the job is on its final retry and avoid re-raising on this line, I think it could accomplish what we're after.

The code for the check of the logs is in [here](https://github.com/18F/identity-devop/blob/471046d2dd506eb95eebd11d7940932feb82abb8/terraform/app/cloudwatch-filters-idp.tf#L222-L226) and ActiveJob should expose the retry information in attributes specified [here](https://github.com/rails/rails/blob/main/activejob/lib/active_job/core.rb#L28-L35)